### PR TITLE
Upgrade tracing and opentelemetry crates

### PR DESCRIPTION
**Pull request by nikurt**

### CODE REVIEW

**Review by matklad — APPROVED** [review]
LGTM, but let's avoid touching Cargo.tomls where we don't need to. Or you might go ahead and do https://github.com/near/nearcore/pull/7687 for tracing

File: chain/client/Cargo.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ actix = "0.13.0"
 actix-rt = "2"
 futures = "0.3"
 chrono = { version = "0.4.4", features = ["serde"] }
-tracing = "0.1.13"
+tracing = "0.1.36"
```
[the remark this answers is not in the document]
Comment (nikurt) [reply chain/client/Cargo.toml]: Thank you, didn't know that.

File: Cargo.lock [hunk 2]
```diff
@@ -453,6 +453,49 @@ dependencies = [
  "tokio",
 ]
 
+[[package]]
+name = "axum"
```
Comment (matklad) [inline Cargo.lock:457]: Hm, did I forget to comment here? 

It is notable that we now ship both axum and acitx-web. That's... unfortunate, but, looking at tonic, it probably has a good reason to use axum. 

On the postigive side, this gives us an argumet for relpacing actix-web with axum some day :)

Comment (nikurt) [reply Cargo.lock]: Removed, thanks

**Pull request merged**